pkg/restore/service_action.go

```diff
@@ -55,6 +55,7 @@ func (a *ServiceAction) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 
 	if service.Spec.ClusterIP != "None" {
 		service.Spec.ClusterIP = ""
+		service.Spec.ClusterIPs = nil
```

Comment:
I noticed that there was already an issue opened for this problem: #4019. The docs state that `ClusterIPs` can have at most 2 values in a dualstack environment. Do we need to check the contents of `ClusterIPs` in addition to this check for `ClusterIP`? @sseago What do you think?

Reply:
Per my understanding, in a single-stack environment, the `ClusterIPs` contains only one element and the element value is the same with `ClusterIP`; in a dual-stack environment, the `ClusterIPs` could hold two elements and the value of the first element is the same with `ClusterIP`. The `ClusterIP` and `ClusterIPs` play the same role.
The code `service.Spec.ClusterIP != "None" {} ` aims to clear the IP address of the service, no matter it's in single-stack or dual-stack environment.

Correct me if I'm wrong @sseago

Reply:
Ah, that makes sense - thanks for the explanation!

Reply:
My original plan with this was to empty `ClusterIPs` when `ClusterIPs` did not have "None" in its first element. However, thinking about it again, when `ClusterIPs` has `None`, `ClusterIP` should also have `None`, so this is a simpler approach. It would be good to confirm in a dual-stack environment what the values of these two fields are both for services with `None` and for those with actual IP addresses, though.